modules/olm-updating-operator-catalog-image.adoc

```diff
@@ -33,6 +33,9 @@ For example, for the `podman` CLI:
 ----
 $ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
 ----
+* For catalogs from private namespaces on link:https://quay.io[quay.io] that your
+account has access to, link:https://quay.io/user/adellape?tab=settings[generate an encrypted password]
+for use with the `--auth-token` flag.
```

Comment:
This is not the correct way to get an auth token. The user needs to use `curl` and `jq` to extract the `token` from the quay login API, see here: https://github.com/operator-framework/operator-courier#authentication
And then put the result in quotes, so the format is:

`--auth-token "${AUTH_TOKEN}"`